src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -756,7 +769,9 @@ function initializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
   // TODO(samstern): This should be done for RTDB as well but it's hard
   // because the convention in prod is subdomain not ?ns=
   if (frb.emulators.firestore) {
-    process.env.FIRESTORE_URL = `${frb.emulators.firestore.host}:${frb.emulators.firestore.port}`;
+    process.env.FIRESTORE_URL = `http://${frb.emulators.firestore.host}:${
```

Comment:
Not having `http://` here was a straight-up bug but we missed it because we couldn't test this at all until the Firestore emulator's recent token change landed.